lms/djangoapps/badges/events/course_meta.py

```diff
@@ -0,0 +1,78 @@
+"""
+Events which have to do with a user doing something with more than one course, such
+as enrolling in a certain number, completing a certain number, or completing a specific set of courses.
+"""
+
+from badges.models import CourseEventBadgesConfiguration, BadgeClass
+from badges.utils import requires_badges_enabled
+
+
+def award_badge(config, count, user):
+    """
+    Given one of the configurations for enrollments or completions, award
+    the appropriate badge if one is configured.
+
+    config is a dictionary with integer keys and course keys as values.
+    count is the key to retrieve from this dictionary.
```

Comment:
unless each key in the dictionary, being named with a unique integer, contains that corresponding number of course keys as its value?